tests/integration/test_raw_persistence.py

```
@@ -0,0 +1,305 @@
+            ingestion_id=ingestion_id
+        ).first()
+        
+        assert raw_record is not None
+        assert raw_record.source_type == "text"
+        assert raw_record.content_type == "text"
+        assert raw_record.raw_payload["content"] == payload["content"]
+        assert raw_record.metadata["tags"] == ["test", "python"]
+        assert raw_record.processed == False
+        assert raw_record.file_data is None
+    
+    def test_duplicate_ingestion_returns_409(self, client, test_db_session):
```

Ruff (0.14.11)

Unused method argument: `client`

(ARG002)